amundsen_application/static/js/config/config-types.ts

```diff
@@ -18,6 +19,7 @@ export interface AppConfig {
 }
 
 export interface AppConfigCustom {
+  badges?: BadgeConfigMap;
```

Comment:
nit: Do you intended to introduce a new naming convention for all configurations that are maps? Or was this just from a naming collision. If it was just to have some unique names I'd be fine with the top level configuration still being named `BadgeConfig` while the nested one is `BadgeStyleConfig`

Reply:
It was to avoid naming collision. I got stuck with `BadgesConfig`. Will fix.